tests/ClassTest.php

```diff
@@ -570,7 +570,6 @@ class ReflectionGenerator {
             'extendException' => [
                 '<?php
                     class ME extends Exception {
-                        /** @var string */
```

Comment:
Deleted this because it registers as unequal since the parent property is not `fromDocBlock`.